package.json

```diff
@@ -28,7 +28,6 @@
     "@folio/eholdings": ">=1.1.0",
     "@folio/erm-comparisons": ">=1.0.0",
     "@folio/erm-usage": ">=1.0.3",
-    "@folio/export-manager": ">=1.0.0",
```

Comment:
I think there is no reason to remove it from package.json, stripes config is enough to disable it on envs

Reply:
@NikitaSedyx 

I'm no expert in FOLIO's front end. Why would I want it included in the bundle if it is not going to be enabled?

Reply:
because it will be enabled later, don't see reason to change lines several times :)

https://github.com/folio-org/ui-export-manager/pull/16 to not disable UI module from envs